packaging/cockpit-machines.spec.in

```diff
@@ -35,7 +35,11 @@ Requires: cockpit-bridge >= 215
 %if 0%{?suse_version}
 Requires: libvirt-daemon-qemu
 %else
-Requires: libvirt-daemon-kvm
+Requires: libvirt-daemon-driver-qemu
+Requires: (qemu-kvm-core or libvirt-daemon-kvm)
+Requires: (libvirt-daemon-driver-storage-core if virt-install)
+Requires: (libvirt-daemon-driver-interface if virt-install)
+Suggests: libvirt-daemon-kvm
```

Comment:
This line doesn't do (in Fedora) what you might think.

In Debian/Ubuntu, this would cue the package manager to provide a message to the user about other packages they might want to install. In Fedora, `Suggests:` is only used to predispose the dependency solver towards a particular package when more than one might provide the same functionality. If the dependency isn't resolved firmly in another way (alternative already installed or pulled in by something else in the transaction), the package from the `Suggests:` directive will be installed.

Reply:
Thanks @sgallagher -- then I figure I'd rather take it out again, otherwise we are back to installing unwanted clutter.